docs/.vitepress/contributors.ts

```diff
@@ -32,12 +37,26 @@ export const contributors = (contributorNames as string[]).reduce((acc, name) =>
   acc.push({ name, avatar: contributorsAvatars[name] })
   return acc
 }, [] as Contributor[])
-
 const createLinks = (tm: CoreTeam): CoreTeam => {
-  tm.links = [
-    { icon: 'github', link: `https://github.com/${tm.github}` },
-    { icon: 'twitter', link: `https://twitter.com/${tm.twitter}` },
-  ]
+  tm.links = [{ icon: 'github', link: `https://github.com/${tm.github}` }]
+  if (tm.webtools)
+    tm.links.push({ icon: 'mastodon', link: `https://elk.zone/m.webtoo.ls/@${tm.webtools}` })
+
+  if (tm.fosstodon)
+    tm.links.push({ icon: 'mastodon', link: `https://elk.zone/fosstodon.org/@${tm.fosstodon}` })
+
+  if (tm.hachyderm)
+    tm.links.push({ icon: 'mastodon', link: `https://elk.zone/hachyderm.io/@${tm.hachyderm}` })
+
+  if (tm.discord)
+    tm.links.push({ icon: 'discord', link: tm.discord })
+
+  if (tm.youtube)
+    tm.links.push({ icon: 'youtube', link: `https://www.youtube.com/@${tm.youtube}` })
+
+  if (tm.twitter)
+    tm.links.push({ icon: 'twitter', link: `https://twitter.com/${tm.twitter}` })
```

Comment:
Can we use a `mastodon` prop with a full link as the value here?

Reply:
sure, so like discord one...

Reply:
done